server/webapp/WEB-INF/rails.new/app/assets/javascripts/build_output_observer.js

```diff
@@ -43,7 +43,6 @@ BuildOutputObserver.prototype = {
 
         if (this.is_completed && this.is_output_empty) {
             this.update_page(json);
-            dashboard_periodical_executer.pause();
```

Comment:
Basically here, build_output_observer is pausing the executer, it is not the responsibility of observer to pause it's executor. Ideally executor should decide itself to pause on certain condition, I have moved this call [here] (https://github.com/chopraapooja/gocd/blob/job-status-json-polling-fix/server/webapp/WEB-INF/rails.new/app/assets/javascripts/dashboard_periodical_executer.js#L70-L72)